client/packages/system/src/IndicatorsDemographics/DetailView/utils.tsx

```diff
@@ -1,18 +1,36 @@
 import { NumUtils } from '@common/utils';
-import {
-  DemographicIndicatorFragment,
-  DemographicProjectionFragment,
-} from '../api/operations.generated';
+import { DemographicProjectionFragment } from '../api/operations.generated';
 
 import { GENERAL_POPULATION_ID } from '../api';
 import { HeaderData, Row } from '../types';
+import {
+  InsertDemographicIndicatorInput,
+  UpdateDemographicIndicatorInput,
+} from '@common/types';
+
+export const toInsertIndicator = (
+  row: Row,
+  indexPopulation?: number
+): InsertDemographicIndicatorInput => {
+  return {
+    id: row.id,
+    name: row.name?.trim() === '' ? undefined : row.name,
```

Comment:
will send name as undefined if it is empty spaces, or empty string, or was undefined to begin with